doc/classes/Input.xml

```diff
@@ -103,6 +104,7 @@
 			<argument index="0" name="axis_index" type="int">
 			</argument>
 			<description>
+				Receives a JOY_AXIS_* Enum and returns it's equivalent name as a String
```

Comment:
Couple style changes:
"Receives a **[code]**JOY_AXIS_\*[/code] constant** and returns **its** equivalent name as a **s**tring**.**"